### PR TITLE
Update to v1.0.0 and add QBO analysis and STE lat-lon plots 

### PR DESCRIPTION
**Pull request by hsiangheleellnl**
Two major chnages updated in this merge:
1. QBO analysis: including three types of plots. 
2. O3 STE flux monthly lat-lon plots. 

Other minor updates:
1. ChemDyg inputs on Zenodo and LCRC share folders
2. Example run_script including QBO option
3. Version updates to v1.0.0

### CODE REVIEW

**Comment by hsiangheleellnl** [comment]
@tangq Please review the pull request. Thank you!

File: chemdyg/templates/chemdyg_STE_flux_diags.bash
Comment (tangq) [inline chemdyg/templates/chemdyg_STE_flux_diags.bash]: This map file only works for ne30pg2 files. Should it be a cfg input, so users can specify the mapping files by themselves?

File: chemdyg/templates/chemdyg_STE_flux_diags.bash
Comment (tangq) [inline chemdyg/templates/chemdyg_STE_flux_diags.bash:227]: Multiplying by 12 ignores that different months have different days. Same for other lines, like line 219 above and other places. Should fix this. Maybe in a separate PR after this one is merged.

File: chemdyg/templates/chemdyg_STE_flux_diags.bash
Comment (tangq) [inline chemdyg/templates/chemdyg_STE_flux_diags.bash]: Hard-coded mapping file name will give us trouble. Make it a cfg input.

File: setup.cfg
Comment (tangq) [inline setup.cfg:3]: Good - the version number update was not forgotten.

**Review by tangq — CHANGES_REQUESTED** [review]

File: chemdyg/templates/chemdyg_STE_flux_diags.bash
Comment (hsiangheleellnl) [reply chemdyg/templates/chemdyg_STE_flux_diags.bash:227]: @tangq Different days in different months have been considered in earlier
```dt = monthrange(2001,month[i])[1]*3600*24```
*12 is to covert the unit from per month to per year.

Comment (tangq) [reply chemdyg/templates/chemdyg_STE_flux_diags.bash:227]: Good to know it's been calculated. Would you add some comments to make it clearer?

**Comment by hsiangheleellnl** [comment]
@tangq The changes have been made as recommended. Please review and approve the pull request. Thanks.

**Review by tangq — APPROVED** [review]
I approve this PR if the test is successful.

**Comment by tangq** [comment]
@hsiangheleellnl , just a reminder - we will do a new release for this.

**Pull request merged**
